src/document/DocumentCommandHandlers.js

```diff
@@ -848,6 +859,19 @@ define(function (require, exports, module) {
         return result.promise();
     }
 
+    /**
+     * Checks and handles if auto update is currently in progress
+     */
+    function checkIfAutoUpdateInProgress() {
+        brackets.app.isAutoUpdateInProgress(function (err, isAutoUpdateInProgress) {
+            if (err === 0 && isAutoUpdateInProgress) {
+                CommandManager.trigger(exports.DIRTY_FILESAVE_CANCELLED);
```

Comment:
Use `execute` over `trigger`.

Reply:
Help me understand. The name of the method is `checkIfAutoUpdateInProgress` but you are executing `exports.DIRTY_FILESAVE_CANCELLED` ? Shouldn't this be something like whoever is calling this method, should be returned a promise and the client can then act upon the promise state.

Reply:
I have now placed all the events related to Auto Update, needed in core, in UpdateNotification in order to ensure consistency.
Also, I now chose a better name for checkIfAutoUpdateProgress(). It is actually a handler function. It iinternally calls a function isAutoUpdateInProgress(), which returns a bool, and the wrapper function handles this bool. So, I have renamed the wrapper function to handleIfAutoUpdateInProgress(). 
Please refer to 537430b